src/components/App.js

```diff
@@ -31,11 +34,13 @@ const App = () => (
       <Route exact path="/terms" component={Terms} />
       <Route exact path="/verify/:token" component={VerifyAccount} />
       <Route exact path="/articles/:slug" component={ReadArticle} />
-      <Route exact path="/" component={Home} />
       <Route exact path="/verify/:token" component={VerifyAccount} />
       <Route exact path="/resetPassword/:token" component={ApplyPassword} />
       <Route exact path="/resetPasswordRequest" component={ResetPasswordRequest} />
       <Route exact path="/profile/:username" component={Profile} />
+      <Route exact path="/articles" component={MyArticles} />
+      <Route exact path="/article/new" component={Create} />
+      <Route exact path="/article/edit/:slug" component={Editarticle} />
```

Comment:
Good job on this feature. However, It would make more sense if you order this route like this: `/articles/:slug/edit` or `/article/:slug/edit`. Thanks.

Reply:
@eliemugenzi, I think that path makes more sense cuz it looks like your edit a certain article based on the provided slug. Thanks!